Common/Symbol.cs

```diff
@@ -300,7 +317,7 @@ public Symbol(SecurityIdentifier sid, string value)
         /// </summary>
         public Symbol UpdateMappedSymbol(string mappedSymbol)
         {
-            if (ID.SecurityType == SecurityType.Option)
+            if (ID.SecurityType == SecurityType.Option || ID.SecurityType == SecurityType.FutureOption)
```

Comment:
This shouldn't happen for `SecurityType.FutureOption`?  It should throw an invalid operation exception if so I think